### DISCUSSION TITLE
Actividad 0 - Danise, Luciana

### DISCUSSION THREAD

**Discussion by luDanise** [frlp-utn-ingsoft/catedra#27] [Show and tell]
https://github.com/luDanise/Actividad0-IS.git

**Comment by RodrigoJacznik** [comment]
Buenas,
Probe la app y no funciono. En el ultimo commit borraste la función `vote`. Por favor arreglalo para aprobar el tp.
Los mensajes de los commits están muy bien. 
Saludos

**Reply by luDanise** [reply]
Hola Rodrigo, ahí agregué la función faltante. Debo corregir algo más para que funcione la app?

**Reply by RodrigoJacznik** [reply]
Hay otro error.
Creo que esta linea https://github.com/luDanise/Actividad0-IS/blob/7122493e3362f6510bcfb58a01fbc0bf488339f8/mysite/polls/templates/polls/index.html#L4 es la problemática. Proba borrandola.
Proba ejecutando la app, intenta que funcione antes de pushear.

**Reply by luDanise** [reply]
Perfecto, ahí la modifiqué y efectivamente era esa linea ya que ahora si puedo correr la app! Muchas gracias.

**Reply by RodrigoJacznik** [reply]
Ahora si 👏
